modules/regular_service_perimeter/variables.tf

```diff
@@ -117,3 +117,15 @@ variable "ingress_policies_dry_run" {
   }))
   default = []
 }
+
+variable "vpc_accessible_services" {
+  description = "A list of [VPC Accessible Services](https://cloud.google.com/vpc-service-controls/docs/vpc-accessible-services) that will be restricted within the VPC Network. Use [\"RESTRICTED-SERVICES\"] to match the restricted services list."
```

Comment:
We should also call out that `*`  is a special value we defined that disables vpc_accessible_services and allows all services.